### DISCUSSION TITLE
Progress Meter Web Interface

### DISCUSSION THREAD

**Discussion by boredofchoosingnames** [outdoorbits/little-backup-box#208] [Ideas]
Afternoon, me again.

I have ordered an OLED screen to go on the LBB, but is there an easy way to see the back-up progress? The Log shows a file or when it is complete, but the IP notifications push that up the list and I can't seem to scroll up the log, is that just me?

Anyway, could we have the information shwon on the screen here:
<img width="201" alt="Screenshot 2023-06-10 163645" src="https://github.com/outdoorbits/little-backup-box/assets/103529663/1d697071-3a77-413f-973a-217ec152eb0c">

Have I missed this somewhere?

Thanks

Frank

**Comment by outdoorbits** [comment]
The automatic scroll of the log implements that you can't really scroll backwards. But you can download the log by clicking the button. Or you configure it to mail the result. But if course, a display is the best option. Did you see the new case including buttons? That's what I absolutely recommend. I'm just an the way back from a trip and enjoyed to quickly use the buttons.

**Reply by boredofchoosingnames** [reply]
Thanks. I don't have time to implement the case before our next trip next week, but I will look into it. I just weent away for the weekend and dowload time with an old laptop was abysmal, so his should improve things.
